src/textfield.vue

```diff
@@ -92,6 +94,16 @@ export default {
   methods: {
     fireInputEvent: function (event) {
       this.$emit('input', event.target.value)
+    },
+    focus () {
+      this.$refs.focusTarget.focus()
+    },
+    focusAndSelect () {
```

Comment:
Well, the docs are still being migrated for v1. I explained somewhere else but I don't remember where, so basically it consists in moving a file in doc/src/partials to src/views